src/backend/renderer/gles/mod.rs

```diff
@@ -366,6 +368,106 @@ pub enum Capability {
     ColorTransformations,
 }
 
+impl Capability {
+    /// Find all capabilities supported by the driver.
+    pub fn supported_capabilities() -> Vec<Capability> {
+        // Ensure OpenGL is initialized.
+        let gl = unsafe { ffi::Gles2::load_with(|s| crate::backend::egl::get_proc_address(s) as *const _) };
+
+        // Get available extensions.
+        let extensions = match GlExtensions::new(&gl) {
+            Ok(extensions) => extensions,
+            Err(_) => return Vec::new(),
+        };
+
+        let mut capabilities = Vec::with_capacity(4);
+
+        // Get maximum supported OpenGL version.
+        let gl_version = version::GlVersion::try_from(&gl).unwrap_or(version::GLES_2_0);
```

Comment:
What if the compositor wants to request a specific GL version?

Reply:
That isn't possible with the current GlesRenderer either, right?

Reply:
Yeah I guess that is an issue we can address later given that we don't allow that at the moment either